### ISSUE TITLE
Is `modsim/test_modsim.py` still a valid unit test

### ISSUE THREAD

**Issue by rioj7** [AllenDowney/ModSimPy#72]
Viewing `modsim/test_modsim.py` in VSC with PyLance shows a lot of errors.

A lot of functions are missing.

An object can't be an instance of a function.

`pint` is not used, and also `UnitStrippedWarning`

**Comment by AllenDowney** [comment]
Yes, the test code is out of date with the current version of modsim.py
